app/lib/frontend/templates/views/layout.mustache

```diff
@@ -23,6 +23,9 @@
   {{#favicon}}
   <link rel="shortcut icon" href="{{{favicon}}}" />
   {{/favicon}}
+  {{#open_search_description}}
```

Comment:
If you are removing it from the root and using `/static/osd.xml`, it can be simply this:

```
<link rel="search" type="application/opensearchdescription+xml" title="Dart packages" href="{{{static_assets.osd_xml}}}"/>
```
(note the closing tag, tests are failing because the lack of it).